nodes/persistence-node/src/services/IpfsGatewayApi.ts

```diff
@@ -65,10 +73,14 @@ export class IpfsGatewayApi {
     app.get('/api/v0/cat', handleError(async (req, res) => {
       const hash = req.query.arg as string;
 
-      const stream = ipfs.cat(hash); 
+      if (hash == null) {
```

Comment:
can this be undefined? maybe use !hash ?

Reply:
Changed both. 

- `!hash` should also cover empty strings, which null check didn't do.
- throwing is used so error ends up at global error handler, which might not be needed here, but I wouldn't avoid that approach in general

Reply:
Also, I don't see the `!hash` change. Maybe you didn't push :D